TVC_MVP/CountyPresenter.swift

```diff
@@ -84,6 +91,23 @@ class CountyPresenter: CountyPresenterProtocol {
         }
     }
 
+    private func generateAlertMessage(county: County) -> String {
+        var newbornString: String = ""
+        var populationString: String = ""
+        let id = Constants.alertTitleBlock + String(county.countyID)
+        let extras = county.countyExtras
+        if extras != nil {
+            if let nb = extras?.newBorns {
+                newbornString = Constants.alertNewborns + String(describing: nb)
```

Comment:
Why do you use `String(describing:)`? `nb` is an `Int`, and `String` has a `init` that accepts this type.